app/javascript/elm/src/LabelsInput.elm

```diff
@@ -51,11 +51,16 @@ update msg model toCmd =
             ( withCandidate str model, Cmd.none )
 
         Add str ->
-            let
-                newModel =
-                    add str model
-            in
-            ( newModel, toCmd (asList newModel) )
+            case str of
```

Comment:
By removing the code above the input sends empty string so we need to guard that

Reply:
Could not find a better way :(

Reply:
Yeah, I remember having that problem as well. I guess we could consider creating a custom type and having `List Label`.

Reply:
Something like a nonempty string?